components/infra-proxy-service/server/servers.go

```diff
@@ -110,12 +88,12 @@ func (s *Server) DeleteServer(ctx context.Context, req *request.DeleteServer) (*
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 
-	UUID, err := uuid.FromString(req.Id)
-	if err != nil {
-		return nil, status.Errorf(codes.InvalidArgument, "invalid server ID: %s", err.Error())
+	if req.Id == "" {
+		s.service.Logger.Debug("incomplete create server request: missing server ID")
```

Comment:
error log instead of debug?

Reply:
if we even need to log...since we're returning the error here we shouldnt need to log?

Reply:
Earlier, somehow I have taken the reference from teams-service. but yeah this something we can get rid of. 

Currently, re-writing validation stuff, it will get removed by PR https://github.com/chef/automate/pull/3540

https://github.com/chef/automate/blob/2a6c0e5f7b2676abacf34ec4cb5b95d19e0506f8/components/infra-proxy-service/server/servers.go#L20-L30